### DISCUSSION TITLE
How do I get started?

### DISCUSSION THREAD

**Discussion by DerMueller** [xournalpp/xournalpp#4719] [Q&A] [answered]
Because asking for a Feature, may never result in actually being a Feature, I wanted to ask how to get started. I already know C/C++. But I don't know how to setup or use CMake. Please help.

**Comment by rolandlo** [comment, marked as answer]
The build process is described in the wiki. Here are the links for [Linux](https://github.com/xournalpp/xournalpp/blob/master/readme/LinuxBuild.md), [Windows](https://github.com/xournalpp/xournalpp/blob/master/readme/WindowsBuild.md) and [MacOS](https://github.com/xournalpp/xournalpp/blob/master/readme/MacBuild.md)
If you get into problems, please ask here or in the [Matrix channel](https://matrix.to/#/#xournalpp_xournalpp:gitter.im).

**Reply by DerMueller** [reply]
Thanks,
But I am getting problems compiling. 
I have mention in Issue #4721 .
What could be the Problem?
